doc/news/_preparation_next_release.md

```diff
@@ -37,8 +37,6 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
 - New Logo
 - INI as new default configuration file format
 - Bindings for Asynchronous I/O
-- <<HIGHLIGHT2>>
```

Comment:
I would prefer to keep these placeholders until short before the release. I hope it encourages people to add their contributions as highlight.